### PR TITLE
Cews show howler

### PR DESCRIPTION
**Pull request by cews7**
@maxglassie @scmountain @janderson16 hey guys! we finished the show page!

### CODE REVIEW

**Comment by maxglassie** [comment]
This looks good. I don't have comments on the code, except that some of the charts show up blank, depending on what data is returned from the API. I've created a card for fixing this as a separate thing to do.

File: Gemfile [hunk 1]
```diff
@@ -29,6 +29,7 @@ gem 'omniauth-twitter'
 gem 'twitter'
 gem "figaro"
 gem 'httparty'
+gem 'rb-readline'
```
Comment (scmountain) [inline Gemfile:32]: You will have to show me what this little bad boy does!

File: app/controllers/howlers_controller.rb [hunk 2]
```diff
@@ -2,4 +2,8 @@ class HowlersController < ApplicationController
   def index
     @howlers = current_user.howlers.order(:created_at).reverse_order
```
Comment (scmountain) [inline app/controllers/howlers_controller.rb:3]: trick way to show latest first

**Review by scmountain — APPROVED** [review]

**Comment by scmountain** [comment]
@maxglassie checkout for merge to master plz

**Pull request merged**